.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: Test
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  smoke:
+    name: "Smoke"
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        provision-type: ["kyma-cli", "kind"]
+        
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        cache: true
+        go-version-file: 'tests/smoke/go.mod'
+        cache-dependency-path: 'tests/smoke/go.sum'
+
+    - name: Set up k3d/kyma for ${{ matrix.provision-type }}
+      if: ${{ matrix.provision-type }} == "kyma-cli"
+      run: |
+        wget -qO - https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | TAG=v5.4.6 bash
+        wget -q https://storage.googleapis.com/kyma-cli-unstable/kyma-linux
```

Comment:
Have you looked into [wget](https://github.com/marketplace/actions/github-action-for-wget) Github action?

Reply:
Its using a 3 year old docker image and is just a shell script for alpine. I figured I just use the inbuilt wget from ubuntu instead until we have to run multi-arch